### DISCUSSION TITLE
Sending the password without encryption in the SignIn function

### DISCUSSION THREAD

**Discussion by ErsinAydogmuss** [supabase/supabase#10438] [General]
When you send the password without no hashing it in the SignIn function, the password appears in the payload section. How can I hide this to the user. If I hashing and send it, I get an invalid password error for supabase because the password is too long.

**Comment by GaryAustin1** [comment]
Can you explain more what you are tying to accomplish?  If even you hash the password, the user can see the hash if they want and use it...
Only the user should be able to see the body of the request if they use developer mode.  Once it leaves the computer it is encrypted as part of the https protocol.

**Reply by ErsinAydogmuss** [reply]
I send it without encryption because Supabase is already hashing, but the unencrypted data as payload appears on the console. Is there a way to circumvent Supabase's encryption?
